File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
       
        if x < 0:
            return False
        
        ans = 0
        original = x
        
        while x > 0:
            rem = x % 10
            x = x // 10
            ans = ans * 10 + rem
        
        if original == ans:
            return True
        
        return False


       
        

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
  if (x < 0) {
            return false;
        }
        int  p=x;
int rem;
int ans=0;
       while(x>0){
         rem=x%10;
        ans=ans*10+rem;
        x=x/10;
       }
       if(p==ans){
        return true;
       }
        return false;
    }
}